Parse/Public/ParseObject.cs

```diff
@@ -1263,7 +1264,7 @@ where item.CanBeSerialized
     /// retrieved and <paramref name="key"/> is not found.</exception>
     /// </summary>
     public T Get<T>(string key) {
-      return (T)ParseClient.ConvertTo<T>(this[key]);
+      return (T)Conversion.ConvertTo<T>(this[key]);
```

Comment:
We could, but I'm not sure its super useful for a public facing API to throw class cast exceptions. Debugging that without symbol information could get confusing fast.